components/eamxx/src/physics/nudging/atmosphere_nudging.cpp

```diff
@@ -281,15 +359,14 @@ void Nudging::run_impl (const int dt)
         T_mid_1d(k).set(below_min,T_mid_ext_1d(0));
         qv_1d(k).set(above_max,qv_ext_1d(num_vert_packs_ext-1));
         qv_1d(k).set(below_min,qv_ext_1d(0));
-        hw_2d(0,k).set(above_max,hw_ext_2d(0,num_vert_packs_ext-1));
-        hw_2d(0,k).set(below_min,hw_ext_2d(0,0));
-        hw_2d(1,k).set(above_max,hw_ext_2d(1,num_vert_packs_ext-1));
-        hw_2d(1,k).set(below_min,hw_ext_2d(1,0));
+        u_1d(k).set(above_max,u_ext_1d(num_vert_packs_ext-1));
```

Comment:
You may want to wrap all this set calls in a `if (above_max.any())` and `if (below_min.any())` respectively.

Reply:
Done